src/expressions.rst

```diff
@@ -375,7 +375,7 @@ The following :t:`[expression]s` are :t:`[place expression]s`:
   expression`,
 
 * :dp:`fls_ya05djl1d154`
-  :t:`[Path expression]s` that resolve to a :t:`binding` or a :t:`static`.
+  :t:`[Path expression]s` that resolve to a :t:`static` or a :t:`variable`.
```

Comment:
Glad to see this change; long ago we discussed this endlessly for the tutorial and then book, and decided on 'variable' there too 👍